res/controllers/Behringer-CMD-MM-1-Advanced-scripts.js

```diff
@@ -502,6 +511,7 @@ CMDMM.init = function () {
 	FUNCTIONS.buttonFromChannelNumber();
 	FUNCTIONS.cycleKnobAssignment();
 	FUNCTIONS.enableSoftTakeover();
+	FUNCTIONS.prepFXUnitSettings();
```

Comment:
Shouldn't this go before `FUNCTIONS.enableSoftTakeover();`?

Reply:
Sorry yeah, I forgot this matters. Can you change it?